App.cs

```diff
@@ -0,0 +1,101 @@
+
+using System.Net.Http.Headers;
+using System.Text;
+
+class App {
+  BattleGround player1;
+  BattleGround player2;
+
+  bool isPlayer1Turn = true;
+
+  public App()
```

Comment:
Fix indents in all files, use Ctrl+K+D in VS for autoformat

Reply:
fixed